apps/cowswap-frontend/src/modules/account/containers/OrdersPanel/index.tsx

```diff
@@ -139,6 +139,8 @@ export function OrdersPanel() {
 
   const handleCloseOrdersPanel = useToggleAccountModal()
 
+  useCloseAccountModalOnNavigate()
```

Comment:
This is used as a "updater" , why do w e add it in the order panel and not in a top updater?

Reply:
Because I only want to run it when the OrderPanel is rendered. There's no need to run this all of the time.

Reply:
But what is the relationship between order pannel, and hidding the account modal?

Reply:
`OrdersPanel` is the component name of the Account modal 🤷 

There's even a TODO about this https://github.com/cowprotocol/cowswap/blob/develop/apps/cowswap-frontend/src/modules/account/containers/OrdersPanel/index.tsx#L131